requirements.txt

```diff
@@ -1,7 +0,0 @@
-affine>=1.3.0
-boto3>=1.2.4
-cligj
-enum34
-numpy>=1.8
-snuggs>=1.2
-setuptools
```

Comment:
@geowurster that sounds good. It's a little tricky since we have the extras sections but if we could find a reliable way to populate those lists from the `requirements*.txt` files, that might be a good option.